frameworks/remix-run/application/src/routes/$langstore.checkout.tsx

```diff
@@ -26,6 +26,8 @@ export const links: LinksFunction = () => {
 };
 
 export default () => {
-    const { isServerSideAuthenticated } = useLoaderData();
+    const { isServerSideAuthenticated } = useLoaderData() as {
```

Comment:
this is not correct you can do it better by changing the loader.

```
export const loader: LoaderFunction = async ({ request }) => {
      return json(something)
};```

instead of that, you can do this:

```
export const loader = async ({ request }: LoaderFunctionArgs) => {
    return json(something)
};

export default () => {
    const something = useLoaderData<typeof loader>();
    ...
};
```

typing will work better this way!